install/helm/README.md

```diff
@@ -0,0 +1,64 @@
+### Open Match Helm Chart Templates
+This directory contains the [helm](https://helm.sh/ "helm") chart templates used to customize and deploy Open Match at ease.
+
+Templates under the `templates/` directory are for the core components in Open Match - e.g. backend, frontend, mmlogic, synchronizor, some security policies, and configmaps are defined under this folder.
+
+Open Match also provides templates for optional components that are disabled by default under the `subcharts/` directory.
+1. `open-match-demo` contains the chart definitions of an example director.
```

Comment:
since we are using the word template elsewhere below, this can be:
_contains the template for a sample director_